taar/recommenders/collaborative_recommender.py

```diff
@@ -49,15 +49,23 @@ def __init__(self, ctx):
         self.logger = self._ctx[IMozLogging].get_logger("taar")
 
         self.model = None
-        self._build_model()
 
     @property
     def addon_mapping(self):
         return self._addon_mapping.get()[0]
 
     @property
     def raw_item_matrix(self):
-        return self._raw_item_matrix.get()[0]
+        val, new_copy = self._raw_item_matrix.get()
+        if new_copy:
+            # Build a dense numpy matrix out of it.
+            num_rows = len(val)
```

Comment:
It looks like Referencing `val` here is causing the CircleCI failure:

>        val, new_copy = self._raw_item_matrix.get()
>       if new_copy:
>            # Build a dense numpy matrix out of it.
>           num_rows = len(val)
> E           TypeError: object of type 'NoneType' has no len()
>
> taar/recommenders/collaborative_recommender.py:62: TypeError`

Reply:
I've reworked this force `self.model` to be None which will cause the `can_recommend` method short circuit and return `False` now.